### PR TITLE
[All] (Mac) Add note about npm run dev-server

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Should address server error issue raised in https://github.com/OfficeDev/office-js-docs-pr/issues/978.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[de225de](https://github.com/OfficeDev/office-js-docs-pr/pull/1040/commits/de225deaefc940de8e024eb9a2723986b8e46c8a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905281858483629-1040%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=855aV2uzHrc15D0bpYOI2jJhNBCzVE7GsF8pkY%2BVTEk%3D&st=2019-05-28T18%3A55%3A19Z&se=2019-06-28T19%3A00%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9b7672a](https://github.com/OfficeDev/office-js-docs-pr/pull/1040/commits/9b7672a2b41e526bffd533fc8b867c12a00f7b2b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905281904078016-1040%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pkb%2F7ewzg7Q5EIAYvJKo0oeVFeUs%2FJhXhVkcPKiafjs%3D&st=2019-05-28T19%3A00%3A34Z&se=2019-06-28T19%3A05%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT Can you & I chat about these changes before this PR is merged? I think there may be a cleaner way of adding clarification, based upon comments in #978. (@davidchesnut FYI - you may want to wait a bit before reviewing this PR, until Elizabeth and I have a chance to chat about it.) Thanks!

**Comment by ElizabethSamuel-MSFT** [comment]
Swapped reviewers

**Comment by o365devx** [comment]
Docs Build status updates of commit _[54e895d](https://github.com/OfficeDev/office-js-docs-pr/pull/1040/commits/54e895d2371e7535e021e81e594f9e41d856b4bc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905282337344478-1040%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3l%2BwIeb%2Ft0E2rzCCkOakdT2s29w7kTtxD0aTX9Qq4Js%3D&st=2019-05-28T23%3A33%3A31Z&se=2019-06-28T23%3A38%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f9c34fe](https://github.com/OfficeDev/office-js-docs-pr/pull/1040/commits/f9c34fed6f4f0b52f5a7d2fae98a4bd2d25db309)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905282340296306-1040%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hhxU4oQOAZwjE0mYiS%2BbhcoTOnKi6FoxSmf8ZY4qKjQ%3D&st=2019-05-28T23%3A36%3A34Z&se=2019-06-28T23%3A41%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/quickstarts/project-quickstart.md [hunk 1]
```diff
@@ -92,15 +92,15 @@ Office.context.document.getSelectedTaskAsync(
 
 ## Try it out
 
-1. Start the local web server by running the following command:
+> [!NOTE]
+> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run one of the following commands, accept the prompt to install the certificate that the Yeoman generator provides.
```
Comment (kbrandl) [inline docs/quickstarts/project-quickstart.md]: Good catch on the fact that the Mac note is not needed in this article!

Now that there's just one command to run, should change "one of the following commands" -> "the following command" in this note.

Comment (ElizabethSamuel-MSFT) [reply docs/quickstarts/project-quickstart.md]:
```suggestion
> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
```

File: docs/quickstarts/project-quickstart.md [hunk 2]
```diff
@@ -92,15 +92,15 @@ Office.context.document.getSelectedTaskAsync(
...
+
+1. Run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
```
Comment (ElizabethSamuel-MSFT) [inline docs/quickstarts/project-quickstart.md]:
```suggestion
1. Run the following command in the root directory of your project. When you run this command, the local web server will start.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5f40476](https://github.com/OfficeDev/office-js-docs-pr/pull/1040/commits/5f40476b2a32f40705cee9751349e3ffc245b479)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905291803345499-1040%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FIYnCej9Xek%2FMpdbSEBSmxMHTNTjKKLaT9xFJhNNkps%3D&st=2019-05-29T17%3A59%3A32Z&se=2019-06-29T18%3A04%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5f40476](https://github.com/OfficeDev/office-js-docs-pr/pull/1040/commits/5f40476b2a32f40705cee9751349e3ffc245b479)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-mac-note/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905291803477673-1040%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=32g9GD95LoCymTWfljkJMp%2BIMuV724A3KfpH0UbNyBM%3D&st=2019-05-29T18%3A00%3A41Z&se=2019-06-29T18%3A05%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**